courses/management/commands/import_courserun.py

```diff
@@ -79,6 +80,13 @@ def add_arguments(self, parser) -> None:
             nargs="?",
         )
 
+        parser.add_argument(
+            "--block_countries",
+            type=str,
+            help="Comma separated list of countries to block enrollments.",
```

Comment:
Does this work for names or 2-letter ISO codes? We should include an example in the help message.

Reply:
It works for both. It looks for the name and if not found then it looks for the code. I will update the docs with an example.